src/components/pages/Landing/Landing.tsx

```diff
@@ -98,16 +97,25 @@ export const Landing = ({ view }: { view?: ViewType }) => {
         !isSubgraphSupported
     );
 
+    useEffect(() => {
```

Comment:
Since there is no dynamic change of modes, I'm wrapping this in a useEffect and calling it in Landing